syft/pkg/cataloger/cpe.go

```diff
@@ -37,14 +37,22 @@ var productCandidatesByPkgType = candidateStore{
 }
 
 var cpeFilters = []filterFn{
-	// nolint: goconst
 	func(cpe pkg.CPE, p pkg.Package) bool {
 		// jira / atlassian should not apply to clients
-		if cpe.Vendor == "atlassian" && cpe.Product == "jira" && strings.Contains(p.Name, "client") {
-			return true
+		if cpe.Product == "jira" && strings.Contains(strings.ToLower(p.Name), "client") {
```

Comment:
nit: Now that we're amassing some nontrivial, specialized rules in our list of `filterFn`, it would be nice to codify these as named functions that get referenced here in this list. But I'm making this a "nit" due to the context of our CPE implementation future

Reply:
agreed with the long term approach, and held off for similar considerations 👍